spec/observables/dom/ajax-spec.ts

```diff
@@ -1154,9 +1167,7 @@ describe('ajax', () => {
         async: true,
         body: undefined,
         crossDomain: true,
-        headers: {
-          'x-requested-with': 'XMLHttpRequest',
-        },
+        headers: {},
```

Comment:
This hopefully helps understand why I had to update many tests. It shows the bad behaviour in 4 lines - `crossDomain` is `true` but at the same time we were expecting the header that should never be sent when `crossDomain` is `true`